src/lib/browser/Browser.ts

```diff
@@ -55,13 +55,21 @@ export class Browser {
     const start = Date.now();
     try {
       const browser = this.#engine === 'firefox' ? firefox : chromium;
+      let executablePath = undefined;
+
+      if (process.env.USE_LATEST_CHROME) {
+        executablePath =
+          this.#engine === 'chromium' ? '/usr/bin/google-chrome' : undefined;
```

Comment:
We discussed about the reasons offline, it can make sense. My only problem is that now that it's build systematically with `USE_LATEST_CHROME`, if you run it without the argument, it will try to use Chromium, which is not downloaded, so it will no work.